_data/hipaa-landing-page-tabs-ref-arch-code.yml

```diff
@@ -0,0 +1,167 @@
+  content: |
+    <figure>
+      <pre>
+      <code class="language-hcl">
+        ########################################
+        # EKS Cluster Configuration in Terraform
+        ########################################
+
+        # ---------------------------------------------------------------------------------------------------------------------
+        # CONFIGURE OUR KUBERNETES CONNECTIONS
+        # Note that we can't configure our Kubernetes connection until EKS is up and running, so we try to depend on the
+        # resource being created.
+        # ---------------------------------------------------------------------------------------------------------------------
+
+        # The provider needs to depend on the cluster being setup.
+        provider "kubernetes" {
+          load_config_file       = false
+          host                   = data.template_file.kubernetes_cluster_endpoint.rendered
+          cluster_ca_certificate = base64decode(data.template_file.kubernetes_cluster_ca.rendered)
+
+          # EKS clusters use short-lived authentication tokens that can expire in the middle of an 'apply' or 'destroy'. To
+          # avoid this issue, we use an exec-based plugin here to fetch an up-to-date token. Note that this code requires a
+          # binary—either kubergrunt or aws—to be installed and on your PATH.
+          exec {
+            api_version = "client.authentication.k8s.io/v1alpha1"
+            command     = var.use_kubergrunt_to_fetch_token ? "kubergrunt" : "aws"
+            args = (
+              var.use_kubergrunt_to_fetch_token
+              ? ["eks", "token", "--cluster-id", module.eks_cluster.eks_cluster_name]
+              : ["eks", "get-token", "--cluster-name", module.eks_cluster.eks_cluster_name]
+            )
+          }
+        }
+
+        # Workaround for Terraform limitation where you cannot directly set a depends on directive or interpolate from resources
+        # in the provider config.
+        # Specifically, Terraform requires all information for the Terraform provider config to be available at plan time,
+        # meaning there can be no computed resources. We work around this limitation by creating a template_file data source
+        # that does the computation.
+        # See https://github.com/hashicorp/terraform/issues/2430 for more details
+        data "template_file" "kubernetes_cluster_endpoint" {
+          template = module.eks_cluster.eks_cluster_endpoint
+        }
+
+        data "template_file" "kubernetes_cluster_ca" {
+          template = module.eks_cluster.eks_cluster_certificate_authority
+        }
+          </code>
+          </pre>
+          <figure>
+- id: infra-docker
+  title: Docker
+  content: |
+    <figure>
+      <pre>
+      <code class="language-docker">
+        #############
+        # Dockerfile
+        #############
+
+        FROM python:3.8-alpine AS with-secrets-python
+        MAINTAINER Gruntwork <info@gruntwork.io>
```

Comment:
We may want a Docker container for an app, not a CI / CD system here... E.g., https://github.com/gruntwork-io/aws-sample-app/blob/master/docker/Dockerfile#L53-L105

Reply:
Done in 40f48e2f